_data/wai-course-list/submissions/3d230470-ffd7-11ed-91f3-af9c47425e87.json

```diff
@@ -0,0 +1,96 @@
+{"submitter_name": "Access42",
+    "submitter_email": "formations@access42.net",
+    "title": "RGAA : auditer l’accessibilité numérique des sites web (RGAA: digital accessibility testing for websites)",
+    "provider": "Access42",
+    "country": [
+      "fr"
+    ],
+    "description": "This training course provides initial skills and knowledge to review the compliance of a website with the RGAA (French accessibility framework): find accessibility issues, write proposals for corrections, prioritize issues and manage exemptions.\\r\\nYou will be provided with tools and methodological elements to help you identify accessibility errors",
```

Comment:
\\r\n is not rendered properly, one back slash needs to be removed i think ! Thank you !